### PR TITLE
fix(ios): properly set some properties

### PR DESCRIPTION
**Pull request by RaulPROP**
- `birthday` was set only if the contact had an image.
- `organizationName` was set only if the contact had an image.
- `organizationRole` was set only if the contact had an image and an `organizationName`.

### CODE REVIEW

File: Rea.chCapacitorContacts.podspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name = 'CapacitorCommunityContacts'
+  s.name = 'Rea.chCapacitorContacts'
```
[the remark this answers is not in the document]
Comment (RaulPROP) [reply Rea.chCapacitorContacts.podspec:6]: Sorry, I was modifying the fork to implement a solution while this was in revision without noticing the branch.

Tomorrow morning I will properly do the rollback to just the fixes.

Sorry for the inconvenience.

[the remark this answers is not in the document]
Comment (RaulPROP) [reply Rea.chCapacitorContacts.podspec:6]: Actually, you can just close this merge and I will open one tomorrow morning with just the fixes if you prefer that.

[the remark this answers is not in the document]
Comment (reslear) [reply Rea.chCapacitorContacts.podspec:6]: so, no problem, notify is complete 😀

**Comment by RaulPROP** [comment]
Revert complete @reslear

**Comment by tafelnl** [comment]
A new major of this plugin just got released as a beta version. The plugin is refactored quite a bit. Among others, the API and TypeScript definitions got improved.

It also ships your request: the birthday and organization fields will be returned, independently of a contact's thumbnail photo.

Check out the newly added docs here: https://capacitor-community.github.io/contacts

Mind you that the new version (v3 and v4) are currently still in beta. However, the beta is tested quite extensively and fairly stable. If you're willing to test it out in your project, it would be much appreciated!

**Pull request closed without merging**